frontend/apps/reader/modules/readertoc.lua

```diff
@@ -644,12 +643,29 @@ function ReaderToc:expandParentNode(index)
 end
 
 function ReaderToc:onShowToc()
+    if self.view.inverse_reading_order then
+        BD.invert()
+    end
+
+    local items_per_page = G_reader_settings:readSetting("toc_items_per_page") or self.toc_items_per_page_default
+    local items_font_size = G_reader_settings:readSetting("toc_items_font_size") or Menu.getItemFontSize(items_per_page)
+    local items_with_dots = G_reader_settings:nilOrTrue("toc_items_with_dots")
+
     self:fillToc()
     -- build menu items
     if #self.toc > 0 and not self.toc[1].text then
+        local tmp = TextWidget:new{
+            text = "    ",
+            face = Font:getFace("smallinfofont", items_font_size),
+            }
+        local toc_indent = tmp:getSize().w
```

Comment:
`-- Have the width of 4 spaces be the unit of/for indentation`
(More comments to avoid the reader to have to guess and doubt his guess :)

Reply:
Done